audit/projects/k8s-release/buckets/k8s-release-eu/metadata.txt

```diff
@@ -0,0 +1,70 @@
+gs://k8s-release-eu/ :
```

Comment:
Same thing, manually created for https://github.com/kubernetes/k8s.io/issues/2396